guides/common/modules/proc_disabling-dns-dhcp-tftp-for-unmanaged-networks.adoc

```diff
@@ -15,9 +19,11 @@ However, {Project} does not remove the back-end services on the operating system
 --foreman-proxy-tftp false
 ----
 
-. In the {ProjectWebUI}, navigate to *Infrastructure* > *Subnets* and select a subnet.
+. For every subnet where the {SmartProxy} is set as a DHCP, DNS, or TFTP proxy, disable the proxies:
+
+.. In the {ProjectWebUI}, navigate to *Infrastructure* > *Subnets* and select a subnet.
 
-. Click the *{SmartProxies}* tab and clear the *DHCP {SmartProxy}*, *TFTP {SmartProxy}*, and *Reverse DNS {SmartProxy}* fields.
+.. Click the *{SmartProxies}* tab and clear the *DHCP {SmartProxy}*, *TFTP {SmartProxy}*, and *Reverse DNS {SmartProxy}* fields.
```

Comment:
```suggestion
.. On the *{SmartProxies}* tab, clear the *DHCP {SmartProxy}*, *TFTP {SmartProxy}*, and *Reverse DNS {SmartProxy}* fields.
```

suggestion (non-blocking): "On the X tab, click Y" is more common throughout the docs.

Note: I also checked the order. :+1: